system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -909,11 +909,7 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 /// Migrations to apply on runtime upgrade.
-pub type Migrations = (
-	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
-	pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
-	InitStorageVersions,
```

Comment:
maybe also remove `struct InitStorageVersions` defined and implemented below

(same for other system paras)

Reply:
Done